backend/src/api/candidates.js

```diff
@@ -38,10 +38,40 @@ router.post(
 )
 
 router.get(
-  '/:candidateId',
+  '/initialize-status',
   errorWrap(async (req, res) => {
-    const candidate = await Candidate.findById(req.params.candidateId)
-    res.json({ result: candidate })
+    await Candidate.update({}, { status: 'pending' }, { multi: true }, err => {
+      if (err) {
+        console.log(err.message)
+      }
+    })
+    res.send('Set all status to pending')
+  })
+)
+
+router.get(
+  '/initialize-year',
+  errorWrap(async (req, res) => {
+    const candidates = await Candidate.find()
+    const years = ['freshman', 'sophomore', 'junior', 'senior']
```

Comment:
the application form has years by graduation semester eg: `Spring 2019`, `Fall 2018`. Let's use that instead

Reply:
But those constantly change